src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -108,9 +116,16 @@ def _store_probabilities_in_bigquery(data):
 
   for row in data:
     bigquery_row = {
-        'strategy_name': row['strategy'],
-        'probability': row['bandit_weight'],
-        'run_count': row['run_count']
+        'strategy_name':
+            row['strategy'],
+        'probability_medium_temperature':
+            row['bandit_weight_medium_temperature'],
+        'probability_low_temperature':
+            row['bandit_weight_medium_temperature'],
+        'probability_high_temperature':
```

Comment:
These three distributions in bigquery are all initialized based on the initial temperature. This will be changed after we have one day of experimental data in our system.

Reply:
Please leave a TODO comment to make this obvious from the code.